libqfieldsync/offline_converter.py

```diff
@@ -102,7 +103,9 @@ def __init__(
         # elipsis workaround
         self.trUtf8 = self.tr
 
-        self.export_folder = Path(export_folder)
+        self.export_folder = Path(export_filename).parent
+        self._export_filename = Path(export_filename).stem
+        self._export_title = export_title
```

Comment:
You need to pass this to `convert_to_offline()` method of `BaseOffliner` instances.

Reply:
Noted!